### PR TITLE
DOCS-1130: Test images classifications

### PR DESCRIPTION
**Pull request by npentrel**
https://docs-test.viam.dev/bc0b11dfce4fa38eca02bfc744504d697db65c1d/public/services/vision/classification/#existing-images-in-the-cloud

### CODE REVIEW

File: docs/services/vision/classification.md [hunk 1]
```diff
@@ -88,7 +88,26 @@ Proceed to [test your classifier](#test-your-classifier).
...
 ## Test your classifier
 
-You can test your classifier with [live camera footage](#live-camera-footage) or [existing images](#existing-images).
+You can test your classifier with [existing images in the Viam app](#existing-images-in-the-cloud), [live camera footage](#live-camera-footage), or [existing images on a computer](#existing-images-on-your-machine).
+
+### Existing images in the cloud
+
+{{< alert title="Note" color="note" >}}
+
+Your classifier must have been trained or uploaded after the September 19, 2023.
```
Comment (andf-viam) [inline docs/services/vision/classification.md:97]:
```suggestion
The feature is only available for classifiers that were uploaded after September 19, 2023.
```

Unless the timestamp of training is also considered in the availability of this feature.

File: docs/services/vision/classification.md [hunk 2]
```diff
@@ -88,7 +88,26 @@ Proceed to [test your classifier](#test-your-classifier).
...
+
+{{< /alert >}}
+
+{{<gif webm_src="/services/vision/mug-classifier.webm" mp4_src="/services/vision/mug-classifier.mp4" alt="A classification model run against an image containing a mug." max-width="250px" class="alignright">}}
+
+If you have images stored in the [Viam cloud](/manage/data), you can run your classifier against your images in the app.
```
Comment (andf-viam) [inline docs/services/vision/classification.md:103]:
```suggestion
If you have images stored in the [Viam cloud](/manage/data), you can run your classifier against your images in the [Viam app](https://app.viam.com/).
```

Let's link directly here; Step 1 links to a how-to, not to the target UI. So let's have the target UI link here.

File: docs/services/vision/classification.md [hunk 3]
```diff
@@ -88,7 +88,26 @@ Proceed to [test your classifier](#test-your-classifier).
...
+
+1. Navigate to the [Data tab](/manage/data/view/) and click on the **Images** subtab.
+2. Click on an image to open the side menu,select the **Actions** tab in the data tab.
```
Comment (andf-viam) [inline docs/services/vision/classification.md:106]:
```suggestion
2. Click on an image to open the side menu, and select the **Actions** tab under the **Data** tab.
```

**Review by andf-viam — APPROVED** [review]
LGTM, just some small language suggestions. Thank you!

File: docs/services/vision/classification.md [hunk 4]
```diff
@@ -88,7 +88,26 @@ Proceed to [test your classifier](#test-your-classifier).
...
+
+{{< alert title="Note" color="note" >}}
+
+The feature is only available for classifiers that were uploaded after September 19, 2023.
+
+{{< /alert >}}
+
+{{<gif webm_src="/services/vision/mug-classifier.webm" mp4_src="/services/vision/mug-classifier.mp4" alt="A classification model run against an image containing a mug." max-width="250px" class="alignright">}}
+
+If you have images stored in the [Viam cloud](/manage/data), you can run your classifier against your images in the [Viam app](https://app.viam.com/).
```
Comment (npentrel) [inline docs/services/vision/classification.md:103]:
```suggestion
If you have images stored in the [Viam cloud](/manage/data/), you can run your classifier against your images in the [Viam app](https://app.viam.com/).
```

**Pull request merged**